### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by RohanSahana**
Increased Readability and corrected all misspelt words

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -60,7 +60,7 @@
 ## Bug Fixes
 
 - Fixed custom domain not setting correct domain
-- Fixed wrong SDK method type in avatars browser route 
+- Fixed wrong SDK method type in avatars browserrouter
```
Comment (eldadfux) [inline CHANGES.md:63]: This was correct before the change

File: CHANGES.md [hunk 2]
```diff
@@ -106,7 +106,7 @@
 ## Breaking Changes
 
 - Updated all the REST API query params to be in camelCase
-- Normalized locale phone codes response body
+- Normalized local phone codes response body
```
Comment (eldadfux) [inline CHANGES.md:109]: This was correct.

**Comment by spielers** [comment]
@RohanSahana  I request you to make changes to your 0.7.x branch and then make PR to 0.7.x branch.
Now you have created changes in your master branch and created PR to the 0.7.x 

best,
spielers

File: CHANGES.md [hunk 3]
```diff
@@ -117,7 +117,7 @@
 
 ## Bug Fixes
 
-- Fixed bug where multiple unique attribute were allowed
+- Fixed bug where multiple unique attribute was allowed
```
Comment (eldadfux) [inline CHANGES.md:120]: Should be: `Fixed bug where multiple unique attributes were allowed`

File: CHANGES.md [hunk 4]
```diff
@@ -157,7 +157,7 @@
 - File upload route (POST /v1/storage/files) now accept a single file per request
 - Added ENV vars to change system email sender name and address 
 - Usage for requests made by project admin in the console are not counted as API usage
-- Added ENV var to change default file upload size limit. New default value is 100MB
+- Added ENV var to change default file upload size limit. New default value is 100MB.
```
Comment (eldadfux) [inline CHANGES.md:160]: Can skip this dot

File: CHANGES.md [hunk 5]
```diff
@@ -179,15 +179,15 @@
 
 ## Features
 
-- Added 5 new locales for locale service and email templates (is, ml, th, fo, ph, pn)
+- Added 5 new locales for local service and email templates (is, ml, th, fo, ph, pn)
```
Comment (eldadfux) [inline CHANGES.md:182]: This was correct

File: CHANGES.md [hunk 6]
```diff
@@ -179,15 +179,15 @@
...
 - 2 stage Docker build
 - Limit HTTP origin check only to browser integrations
 - Updated new Brexit date to 31-01-2020
 - Added a version number to sign in and signup pages for easier debugging of issues
 - Preparation for adding SameSite cookie option support
 - Using native Docker volumes for setup for better cross-platform support and easier management of read/write permissions
 - Added support for custom SSL certificates without needing to set a proxy
-- Added project UID validation check when making an API call. This should help developers to understand our authentication errors better.
+- Added project: UID validation check when making an API call. This should help developers to understand our authentication errors better.
```
Comment (eldadfux) [inline CHANGES.md:190]: This is redundant.

File: CHANGES.md [hunk 7]
```diff
@@ -218,7 +218,7 @@
 
 ## Features
 
-- Added 19 new locales for locale service and email templates (af, ar, bn, cz, hu, hy, jv, ko, lt, ml, no, ru, si, sq, sv, ta, vi, zh-cn, zh-tw)
+- Added 19 new locales for local service and email templates (af, ar, bn, cz, hu, hy, jv, ko, lt, ml, no, ru, si, sq, sv, ta, vi, zh-cn, zh-tw)
```
Comment (eldadfux) [inline CHANGES.md:221]: This was correct

File: CHANGES.md [hunk 8]
```diff
@@ -243,7 +243,7 @@
 - Added option to disable abuse check and rate limits
 - Added input field with the server API endpoint for easy access
 - Added new OAuth providers for Google, Bitbucket, and GitLab
-- Added 15 new locales for locale service and email templates (cat, de, es, fi, fr, gr, hi, id, it, nl, pt-br, pt-pt, ro, tr, ua)
+- Added 15 new locales for local service and email templates (cat, de, es, fi, fr, gr, hi, id, it, nl, pt-br, pt-pt, ro, tr, ua)
```
Comment (eldadfux) [inline CHANGES.md:246]: This was correct

File: CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -11,7 +11,7 @@ appearance, race, religion, or sexual identity and orientation.
```
Comment (eldadfux) [inline CODE_OF_CONDUCT.md:11]: Please revert changes in this file. This is a standard format.

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -54,15 +54,15 @@ If you go to your repository on GitHub, you'll see a `Compare & pull request` bu
 Now submit the pull request and , click on `Create pull request`.
 6. Get a code review approval / reject
 7. After approval, merge your PR
-8. GitHub will automatically delete the branch, after the merge is done. (they can still be restored).
+8. GitHub will autodelete the branch, after the merge is done. (they can still be restored).
```
Comment (eldadfux) [inline CONTRIBUTING.md:57]: redundant.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -54,15 +54,15 @@ If you go to your repository on GitHub, you'll see a `Compare & pull request` bu
...
 
-> If you just want to install Appwrite for day-to-day usage and not as a code maintainer use this [installation guide](https://github.com/appwrite/appwrite#installation).
+> If you just want to install Appwrite for day-to-day usage and not as a code maintainer, use this [installation guide](https://github.com/appwrite/appwrite#installation).
 
-Please note that these instructions are for setting a functional dev environment. If you want to set up an Appwrite instance to integrate into your app, you should probably try and install Appwrite by using the instructions in the [getting started guide](https://appwrite.io/docs/getting-started-for-web) or in the main [README](README.md) file.
+Please note that these instructions are for setting a functional dev environment. If you want to set up an Appwrite instance to integrate into your app, probably try to install Appwrite by using the instructions in the [getting started guide](https://appwrite.io/docs/getting-started-for-web) or in the main [README](README.md) file.
```
Comment (eldadfux) [inline CONTRIBUTING.md:65]: Not sure about this change.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -145,7 +145,7 @@ Although the Appwrite API is a monolithic app, it has a very clear separation of
 
 Each container in Appwrite is a microservice on its own. Each service is an independent process that can scale without regard to any of the other services.
 
-Currently, all of the Appwrite microservices are intended to communicate using the TCP protocol over a private network. You should be aware to not expose any of the services to the public-facing network, besides the public port 80 and 443, who, by default, are used to expose the Appwrite HTTP API.
+Currently, all of the Appwrite microservices are intended to communicate using the TCP protocol over a private network. You should be aware of not expose any of the services to the public-facing network, besides the public port 80 and 443, who, by default, are used to expose the Appwrite HTTP API.
```
Comment (eldadfux) [inline CONTRIBUTING.md:148]: Both version seems off to me

**Pull request closed without merging**